### ISSUE TITLE
questions encountered when the number of wide residual blocks increases

### ISSUE THREAD

**Issue by hythbr** [JiahuiYu/wdsr_ntire2018#21]
Thank you for your sharing. 
When I try to train WDSRa on DIV2k by setting the number of  wide residual blocks as 16,  it works well with the loss of the first 100 iterations being about 15. However, when the number of wide residual blocks  increases to 128, the loss of the first 100 iterations is about 2e8 and the PSNR evaluated on Set5 (x2) is about 18dB . I did not find the reason. So ask you, when the number of blocks increases, how to set the relevant hyperparameters?  Thank you.

**Comment by JiahuiYu** [comment]
When number of blocks increase, set residual scale as 0.1. This trick is introduced in EDSR paper.

**Comment by hythbr** [comment]
Thank you for your reply. I will try.
